homeassistant/components/metoffice/weather.py

```diff
@@ -94,7 +107,7 @@ def visibility(self):
         """Return the platform visibility."""
         _visibility = None
         if hasattr(self.metoffice_now, "visibility"):
-            _visibility = f"{VISIBILITY_CLASSES.get(self.metoffice_now.visibility.value)} - {VISIBILITY_DISTANCE_CLASSES.get(self.metoffice_now.visibility.value)}"
+            _visibility = f"{VISIBILITY_CLASSES.get(self.metoffice_now.visibility.value)} ({VISIBILITY_DISTANCE_CLASSES.get(self.metoffice_now.visibility.value)})"
```

Comment:
Is this now a breaking change in case other users are using this in automations for example?  Why does it need to be changed?

Also, is the `visibility_unit` property method below used anywhere?  I don't see it referenced in this module, or in the weather integration.  This might be a separate PR to clean up.